### PR TITLE
Create team-channel-management.rst

### PR DESCRIPTION
**Pull request by justinegeffen**
New section in onboarding for: https://github.com/mattermost/mattermost-webapp/pull/5469.

### CODE REVIEW

**Comment by justinegeffen** [comment]
Documentation for https://github.com/mattermost/mattermost-webapp/pull/5469.

File: source/deployment/team-channel-management.rst [hunk 1]
```diff
@@ -0,0 +1,106 @@
+Managing Team and Channel Members
```
Comment (wiersgallak) [inline source/deployment/team-channel-management.rst:1]: Is this E20 only currently?

Comment (justinegeffen) [reply source/deployment/team-channel-management.rst:1]: Good question. @thefactremains - can you confirm?

Comment (amyblais) [reply source/deployment/team-channel-management.rst:1]: It's E20.

Comment (justinegeffen) [reply source/deployment/team-channel-management.rst]: Updated.

**Pull request merged**